### DISCUSSION TITLE
simple script for heroku app to generate the session string made by me

### DISCUSSION THREAD

**Discussion by erickythierry** [phaticusthiccy/WhatsAsenaDuplicated#75] [Ideas]
**Hello everybody.** 👋  
thank you for maintaining this project.

I made this simple script to help bot login
everything is done by the website without major problems

repository link below
[asena-qr-gerador](https://github.com/erickythierry/asena-qr-gerador)

I hope it can be useful 👍

![alt text](https://raw.githubusercontent.com/erickythierry/asena-qr-gerador/4d8ffb4f74696f625bb8762d7838a6d93886921c/exemple.png)

**Comment by xmtscf** [comment]
Boa... Agora da pra instalar o bot bem mais rápido...

**Comment by phaticusthiccy** [comment]
You can use heroku client to shut down app after usage, some users don't know to delete their apps. 
Also did u try this previously? 

I haven't checked yet

**Comment by erickythierry** [comment]
**Mod the interface** 
More like Whatsapp Web

![alt text](https://raw.githubusercontent.com/erickythierry/asena-qr-gerador/729c4a2f927a6745fc829b00c4517699532fb94d/exemple.png)
